docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -2008,3 +2008,206 @@ generic object: ALAddress
+    datatype: yesnoradio
+---
+generic object: ALIndividual
+id: has x changed their name again
+question: |
+  Is there another time that ${ x.preferred_name } changed their name?
+fields:
+  - no label: x.previous_names.there_is_another
+    datatype: yesnoradio
+---
+id: birth name
+question: |
+  What was your first legal name?
+subquestion: |
+  Write the name as it appeared on your first birth certificate.
+fields:
+  - First name: users[0].previous_names[0].first
+  - Middle name: users[0].previous_names[0].middle
+    required: False
+  - Last name: users[0].previous_names[0].last
+  - Suffix: users[0].previous_names[0].suffix
+    required: False
+    code: name_suffix()
+---
+id: minor birth name
+question: |
+  What was the minor's first legal name?
+subquestion: |
+  Write the name as it appeared on their first birth certificate.
+fields:
+  - First name: children[0].previous_names[0].first
+  - Middle name: children[0].previous_names[0].middle
+    required: False
+  - Last name: children[0].previous_names[0].last
+  - Suffix: children[0].previous_names[0].suffix
+    required: False
+    code: name_suffix()
+---
+generic object: ALIndividual
+id: x birth name
+question: |
+  What was ${ x.preferred_name }'s first legal name?
+subquestion: |
+  Write the name as it appeared on their first birth certificate.
+fields:
+  - First name: x.previous_names[0].first
+  - Middle name: x.previous_names[0].middle
+    required: False
+  - Last name: x.previous_names[0].last
+  - Suffix: x.previous_names[0].suffix
+    required: False
+    code: name_suffix()
+---
+id: name i
+question: |
+  What was your ${ ordinal(i) } name?
+fields:
+  - First name: users[0].previous_names[i].first
+  - Middle name: users[0].previous_names[i].middle
+    required: False
+  - Last name: users[0].previous_names[i].last
+  - Suffix: users[0].previous_names[i].suffix
+    required: False
+    code: name_suffix()
+---
+id: minor name i
+question: |
+  What was the minor's ${ ordinal(i) } name?
+fields:
+  - First name: children[0].previous_names[i].first
+  - Middle name: children[0].previous_names[i].middle
+    required: False
+  - Last name: children[0].previous_names[i].last
+  - Suffix: children[0].previous_names[i].suffix
+    required: False
+    code: name_suffix()
+---
+generic object: ALIndividual
+id: x i name
+question: |
+  What was ${ x.preferred_name }'s ${ ordinal(i) } legal name?
+fields:
+  - First name: x.previous_names[i].first
+  - Middle name: x.previous_names[i].middle
+    required: False
+  - Last name: x.previous_names[i].last
+  - Suffix: x.previous_names[i].suffix
+    required: False
+    code: name_suffix()
+---
+id: consent of parent 1
+question: |
+  Do you consent to ${ children[0].preferred_name }'s name change?
+fields:
+  - I consent: users[0].consented_to_name_change
+    datatype: yesnoradio
+---
+id: consent of parent 1
+question: |
+  Does ${ users[1] } consent to ${ children[0].preferred_name }'s name change?
```

Comment:
I don't think the consent questions belong in AL like this; they're very specific to a single for or set of forms, and I can't imagine anyone using them without modifying something about them.